application/helpers/expressions/em_manager_helper.php

```diff
@@ -5070,7 +5070,9 @@ private function _UpdateValuesInDatabase($finished = false)
                 $message .= $this->gT("Unable to insert record into survey table"); // TODO - add SQL error?
```

Comment:
this gets overwritten so it can be removed

Reply:
Fixed.